portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/includeLoginNav.vm

```diff
@@ -9,7 +9,12 @@
             #if (${usePortalSearch})
             <sakai-search page-size="${portalSearchPageSize}"></sakai-search>
             #end
-
+            #if (${currentRole} != "" && $roleSwitchState == false)
```

Comment:
I have doubts about this part....I don't see why you're doing this comparison:

$roleSwitchState == false

Just check the currentRole...

 #if (${currentRole} != null)

Reply:
It's because if an instructor user want to see how looks in other role (student or teaching assistant) it will display a failed currentRole

Reply:
Ahhh I seeee, makes sense to me, it's the EXIT link,